Solved the issue of popping effect, animations for icons

Tharunkunamalla opened the pull request:
Now User can hover on the cards.... with user interface and added some styling for the cards so make it look good and added key props for the sentimentChartPreview because triggered an issue and added some piece of code of animation in the index.css
This is my output for the Assigned word

<img width="1690" height="741" alt="image" src="https://github.com/user-attachments/assets/860191d5-5fb5-401c-8e70-a1a98c021fdb" />

closed: #34 

Thank You

IkkiOcean commented:
<img width="390" height="300" alt="Screenshot 2025-07-24 at 9 15 43 PM" src="https://github.com/user-attachments/assets/647e4ce4-d0bc-4231-9c9b-7c861bd0a005" />

you task was: 
- **remove** below emojis

- add bounce to above placed lucide react icons

make requested changes and ping me.

Tharunkunamalla commented:
<img width="1132" height="447" alt="image" src="https://github.com/user-attachments/assets/35c20ec5-3bd6-4bde-9eb5-6d56ec9f3452" />
<img width="1128" height="479" alt="image" src="https://github.com/user-attachments/assets/3db71861-0f2c-4d79-8f97-ae44b8b79289" />
<img width="1119" height="455" alt="image" src="https://github.com/user-attachments/assets/58a055a0-7cad-4196-8798-a31e2424f6b8" />
I have added one new icon for log your mood and removed that icon for that for only one card..... the below icons have some animations... when we hovered
video:

https://github.com/user-attachments/assets/6202cd9a-21b5-4be4-8dc1-a2e1e8508549

check this and say me if this below emojis are not fine then i will remove it permanently 
@IkkiOcean 
Thank you

IkkiOcean commented:
no need to merge conflicts, just do your issue. conflicts are complicated i will do it.

Tharunkunamalla commented:
<img width="1600" height="363" alt="image" src="https://github.com/user-attachments/assets/e8628165-8d0a-4e06-8ec3-41d05c3c2589" />
@IkkiOcean is this ok?
